addon/templates/components/ember-table.hbs

```diff
@@ -18,20 +18,21 @@
 </thead>
 
   <div class="et-tbody-container" data-test-body-container>
-    {{#vertical-collection rows
-      estimateHeight=estimateRowHeight
-      tagName="tbody"
-      class="et-tbody"
+    <tbody class="et-tbody">
+      {{#vertical-collection rows
```

Comment:
`vertical-collection` is now tagless, so we need to create an external `tbody` element and add a `containerSelector`